lib/libc/locale/Symbol.map

```diff
@@ -199,6 +199,7 @@ FBSD_1.3 {
 	__istype_l;
 	__runes_for_locale;
 	_ThreadRuneLocale;
+	__emutls_v._ThreadRuneLocale;
```

Comment:
Hmm, does this belong here or in aarch64?

Reply:
Good question, I think any arch that used emutls would need it as it is the emutls version of _ThreadRuneLocale (that is, I think you get one or the other).  I think in that sense it probably makes sense to live here as emultls isn't really aarch64 specific.  I am hopeful we can kill emutls for morello though at some point?

Reply:
Well, __emutls_get_address is over in the aarch64 Symbol.map

Reply:
Hmm, I guess they should either all be in aarch64 or the generic emutls one should be in gen/Symbol.map.